crates/bevy_color/src/color.rs

```diff
@@ -171,6 +171,18 @@ impl Color {
         })
     }
 
+    /// Creates a new [`Color`] object storing a [`Srgba`] color from a [`u32`] value with an alpha of 1.0.
+    ///
+    /// For example, a value of `0x000000` results in black, and a value of `0xff0000` results in red.
+    pub fn srgb_u32(color: u32) -> Self {
```

Comment:
Would it be a nice addition to also have an `srgba_u32` function since you can fit the four channels in 32 bits?